packages/react/src/components/Dropdown/Dropdown.tsx

```diff
@@ -333,15 +337,16 @@ class Dropdown extends AutoControlledComponent<Extendable<DropdownProps>, Dropdo
     styles: ComponentSlotStylesInput,
     getToggleButtonProps: (options?: GetToggleButtonPropsOptions) => any,
   ): JSX.Element {
+    const { inline, fluid, triggerButton } = this.props
     const content = this.getSelectedItemAsString(this.state.value)
 
     return (
       <Ref innerRef={this.buttonRef}>
-        {Button.create(this.props.triggerButton, {
+        {Button.create(triggerButton, {
           defaultProps: {
             className: Dropdown.slotClassNames.triggerButton,
             content,
-            fluid: true,
+            fluid: fluid || !inline,
```

Comment:
I'm not that sure about this condition; @silviuavram this is about the `Button` component we use for selection `Dropdown`, which should be `fluid` in all cases except `inline`.

However `<Dropdown fluid ../>` should override the `inline` condition anyway, hence the `fluid: fluid || !inline` condition.

I'm thinking to leave just `fluid` at this line and move the logic for `!inline` to the styles, what do you think?

Reply:
I think that we need to extract it to a separate variable `buttonFluid` and leave a comment, we should keep styles simple 🐱

Reply:
it was just one line in styles, I reverted the condition here; don't think we can make it better at this point :(

Reply:
maybe I don't fully understand what is going on here. If the Dropdown is inline, ok, fine, the whole container will be inline. Why does the Button, which is fluid inside that container, should be affected? As long as you handle the whole inline case through the first `<div>` (the container one), all should be exactly as it was before.

please let me know what I am missing :)